src/debug.c

```diff
@@ -2252,7 +2252,7 @@ static void sigsegvHandler(int sig, siginfo_t *info, void *secret) {
 
     bugReportStart();
     serverLog(LL_WARNING,
-        "Redis %s crashed by signal: %d, si_code: %d", REDIS_VERSION, sig, info->si_code);
+        "Server version %s crashed by signal: %d, si_code: %d", SERVER_VERSION, sig, info->si_code);
```

Comment:
If we print the version, we may also want to print the server name. Just an idea.

```suggestion
        SERVER_NAME " %s crashed by signal: %d, si_code: %d", SERVER_VERSION, sig, info->si_code);
```

Reply:
Sure, I wanted to add it one other place as well